go/libkb/secret_store.go

```diff
@@ -129,6 +129,39 @@ func GetConfiguredAccounts(m MetaContext, s SecretStoreAll) ([]keybase1.Configur
 			IsCurrent: username.Eq(currentUsername),
 		}
 	}
+
+	// Get the full names
+
+	uids := make([]keybase1.UID, 0, len(allUsernames))
+	uidMapper := m.G().UIDMapper
```

Comment:
nit: you're just referencing this twice and it's saving like 10 characters in total, I avoid doing stuff like this unless I reference something tens of times

Reply:
It feels ugly to me to be calling a method to get a thing multiple times... maybe I just need to get used to this whole `m.G()` pattern..

Reply:
therefore it's a nit, G() just returns a pointer anyways